### PR TITLE
feat(map): Add basic map in homepage

### LINKED ISSUE #3: Create a basic map page for the homepage
Use [leaflet](https://leafletjs.com/) to create a basic map page that will eventually be populated with markers individual locations in a future ticket.

### PR DESCRIPTION
**Pull request by Zen-cronic**
Resolves #3

- Map created using leaflet and react-leaflet
- default configs based on v1 repo's [configs](https://github.com/CivicTechTO/tRacket-dashboard/blob/6a0a3c82ba72210d81b8b0edde61858039cb70f9/app/src/config.ini)

Current view:

tracket-basic-map

### CODE REVIEW

**Comment by Zen-cronic** [comment]
With clusters and tooltips: 

tracket-basic-map-clusters

File: tracket-react/src/components/map/Map.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  return (
+    <MapContainer center={[MAP_CENTER_LAT, MAP_CENTER_LON]} zoom={ZOOM}  zoomControl={true}>
+      <TileLayer url={LAYER_URL} attribution={LAYER_ATTRIBUTION} />
+
+      <MarkerClusterGroup>
+        {locations.map((l) => {
+          const position: LatLngTuple = [l.latitude, l.longitude];
+          const toolTipActive = l.active
+            ? "Active Location"
```
Comment (Zen-cronic) [inline tracket-react/src/components/map/Map.tsx]: Similar to v1: https://github.com/CivicTechTO/tRacket-dashboard/blob/6a0a3c82ba72210d81b8b0edde61858039cb70f9/app/src/app_components.py#L192-L207

File: tracket-react/src/components/map/Map.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+          return (
+            <Marker position={position} key={l.id}>
+              <Tooltip>
+                <div>
+                  <b>{toolTipActive}</b>
+                  <br></br>
+                  {toolTipLabel}
+                </div>
+              </Tooltip>
```
Comment (Zen-cronic) [inline tracket-react/src/components/map/Map.tsx]: should be a separate component of its own. To be refactored

File: tracket-react/src/main.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
 import './index.css'
+import Map from './components/map/Map'
 
 createRoot(document.getElementById('root')!).render(
   <StrictMode>
-    <div>Hello, World!</div>
+    <Map/>
```
Comment (Zen-cronic) [inline tracket-react/src/main.tsx:8]: TODO: move it inside a parent component after setting up routing

**Comment by Zen-cronic** [comment]
updated look (similar to v1): 

tracket-map-clusters-updated


TODO: 

- [x] Make components out of custom clusters and tooltips
- [x] ~Include the device in Hamilton (id 782239) into initial load view~ Another PR
- [x] ~Set up e2e test with playwright to verify the number of clusters/markers before making further design changes (probably in an another PR)~ Another PR

**Comment by MaanasArora** [comment]
@Zen-cronic Thank you for this--it looks really good! Didn't find any issues. I've approved and merged it.

**Pull request merged**